### PR TITLE
Add note about PlayKeys.playMonitoredFiles

### PR DESCRIPTION
**Pull request by sakthipriyan**
Added line would have saved me few hours. I think this will help people who use standard sbt layout.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @sakthipriyan,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: documentation/manual/gettingStarted/Anatomy.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ dist
 
 ## Default SBT layout
 
-You also have the option of using the default layout used by SBT and Maven. Please note that this layout is experimental and may have issues. In order to use this layout, use `disablePlugins(PlayLayoutPlugin)`. This will stop Play from overriding the default SBT layout, which looks like this:
+You also have the option of using the default layout used by SBT and Maven. Please note that this layout is experimental and may have issues. In order to use this layout, use `disablePlugins(PlayLayoutPlugin)`. Also, add `PlayKeys.playMonitoredFiles ++= (sourceDirectories in (Compile, TwirlKeys.compileTemplates)).value` so that auto reload seemlessly work for twirl templates. This will stop Play from overriding the default SBT layout, which looks like this:
```
Comment (gmethvin) [inline documentation/manual/gettingStarted/Anatomy.md]: The last sentence is now a bit confusing. Maybe you could change to "The default SBT layout looks like this:"

**Comment by benmccann** [comment]
See https://github.com/playframework/playframework/issues/4715

File: documentation/manual/gettingStarted/Anatomy.md (hunk 1 above)
Comment (benmccann) [inline documentation/manual/gettingStarted/Anatomy.md]: Also, seamlessly is misspelled

**Comment by wsargent** [comment]
Created a new PR https://github.com/playframework/playframework/pull/5312 incorporating this note

**Pull request closed without merging**